src/value-types.js

```diff
@@ -81,6 +81,12 @@ function IntegerValue(value) {
         /* remainder with the sign of the divisor */
         return IntegerValue(((value % b) + b) % b);
     };
+    that.pow = function(b) {
+        if (b.getValue() < 0) {
+            throw new Error(value + ' is negative');
```

Comment:
Surely you mean `b.getValue() + ' is negative'`? Otherwise, that would be a *very* confusing error message ^^